### DISCUSSION TITLE
Rest api endpoints are acessible on the localhost but not on the server

### DISCUSSION THREAD

**Discussion by Divyanshu-2022** [community/community#60906] [Apps, API and Webhooks]
### Select Topic Area

General

### Body

![](https://cdn.discordapp.com/attachments/1129397372498223145/1129397895553089566/dcfe5e18-f017-4b1c-a3d8-090dd30a77cd.jpg)
``Hi, As I'm fresher of 4 month into web-development into the industry,
I'm working on a big project that uses cms to update on the website and post the data to the database of mongo-db Atlas to update the database 

as I was working on api to create new endpoints but the problem as i have created the api endpoints they are working fully fine on the localhost and fetching the data fully fine but as i edited the files of server through use of file-zilla then there is error of can't get /the-end-point that are newly created not a single one are working 
kindly suggest me help here its on urgent to be done

kindly find the picture where im fetching the data on local and server
![rn_image_picker_lib_temp_08113595-445f-4143-8dfa-de1f45a3ae78](https://github.com/community/community/assets/104244101/7fa8b048-70ac-46fb-8d96-86227f3ac785)
![dcfe5e18-f017-4b1c-a3d8-090dd30a77cd](https://github.com/community/community/assets/104244101/b0358c8b-bf2b-4b1e-b82b-59fc5335f22b)

**Comment by nitishmaurya85** [comment]
Yes, I know there using
